.github/workflows/spec-parser.yml

```diff
@@ -0,0 +1,35 @@
+name: run spec-parser
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - name: get spec-parser
+        run: |
+          git clone https://github.com/spdx/spec-parser/
```

Comment:
`--depth=1 --single-branch --branch=main` is what I'm usually adding (not that it matters much in this case)